lib/Service/MembersService.php

```diff
@@ -387,8 +389,18 @@ private function verifyIdentContact(&$ident, $type) {
 	 */
 	private function addGroupMembers(Circle $circle, $groupId) {
 
-		$group = OC::$server->getGroupManager()
-							->get($groupId);
+		$groupManager = OC::$server->getGroupManager();
+		$group = $groupManager->get($groupId);
+
+		$user = OC::$server->getUserSession()->getUser();
```

Comment:
should be added as a parameter of the `__construct()`. can you do the same with IGroupManager ?